mautrix_telegram/__main__.py

```diff
@@ -44,6 +45,8 @@
 except ImportError:
     prometheus = None
 
+PORTAL_INACTIVE_THRESHOLD = 2 * 60 # 2 minutes
```

Comment:
This should be configurable IMHO - at least for my personal usage I get telegram events maybe once a week, which I assume could be true for many small EMS hosts, and we don't want a flapping bridge due to little usage. So the default should be very long OR disabled by configuration possibly?

Reply:
+1 I would only have this for large hosts atm.

Reply:
It's become the config option `telegram.liveness_timeout`.